Microsoft.Toolkit.Uwp.SampleApp/Microsoft.Toolkit.Uwp.SampleApp.csproj

```diff
@@ -1564,6 +1574,7 @@
       <Name>Visual C++ 2015 Runtime for Universal Windows Platform Apps</Name>
     </SDKReference>
   </ItemGroup>
+  <ItemGroup />
```

Comment:
><ItemGroup /> [](start = 2, length = 13)

You can remove this empty group

Reply:
Ah, good catch, I had missed it, thanks! 😊

Fixed in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3357/commits/15cecd4b9ea4e47266e51d2a94ae45ce681d8e4d.